packages/bitcore-wallet-service/src/lib/emailservice.ts

```diff
@@ -326,7 +326,7 @@ export class EmailService {
       }
 
       if (_.includes(['NewIncomingTx', 'NewOutgoingTx'], notification.type) && data.txid) {
-        const urlTemplate = this.publicTxUrlTemplate[wallet.chain][wallet.network];
+        const urlTemplate = this.publicTxUrlTemplate[wallet.chain]?.[wallet.network];
         if (urlTemplate) {
```

Comment:
@gabrielbazan7 brought up a good point offline. Let's add an else after this if and log the chain/network when urlTemplate doesn't exist

Reply:
Good plan. Added the logging in the latest commit.